app/models/manual.rb

```diff
@@ -66,4 +66,8 @@ def publication_state
   def published?
     state == "published"
   end
+
+  def withdraw!
```

Comment:
I'd be inclined to make this a non-`!` method, for consistency with the `publish` and `update` methods on this model (even though `SpecialistDocument` has `withdraw!`). It's probably also a good idea to return `self` from this method, again for consistency with the two other methods which modify state here.